pandas/tests/reshape/concat/test_datetimes.py

```diff
@@ -403,6 +403,7 @@ def test_concat_multiple_tzs(self):
         expected = DataFrame({"time": [ts2, ts3]})
         tm.assert_frame_equal(results, expected)
 
+    @pytest.mark.filterwarnings("ignore:Timestamp.freq is deprecated:FutureWarning")
     def test_concat_multiindex_with_tz(self):
```

Comment:
Same here,  what's causing this test to raise a warning? (I don't directly see a `.freq` access)